### PR TITLE
[mysql] Support docker image of ARM chip

### PR DESCRIPTION
**Pull request by minchowang**

### CODE REVIEW

**Comment by minchowang** [comment]
#1367

File: flink-connector-mysql-cdc/src/test/java/com/ververica/cdc/connectors/mysql/testutils/MySqlContainer.java [hunk 1]
```diff
@@ -33,7 +33,7 @@
 @SuppressWarnings("rawtypes")
 public class MySqlContainer extends JdbcDatabaseContainer {
 
-    public static final String IMAGE = "mysql";
+    public static final String IMAGE = "mysql/mysql-server";
```
Comment (leonardBang) [inline flink-connector-mysql-cdc/src/test/java/com/ververica/cdc/connectors/mysql/testutils/MySqlContainer.java:36]: IIUC, only MySQL-8.0 works in Mac with ARM chips. right?

https://hub.docker.com/r/mysql/mysql-server/

Comment (minchowang) [reply flink-connector-mysql-cdc/src/test/java/com/ververica/cdc/connectors/mysql/testutils/MySqlContainer.java:36]: No，`mysql/mysql-server:5.7` also can it.  

image

**Comment by minchowang** [comment]
How can retry run UT checks？ @leonardBang

File: flink-connector-mysql-cdc/src/test/java/com/ververica/cdc/connectors/mysql/testutils/MySqlContainer.java (hunk 1 above)
Comment (minchowang) [reply flink-connector-mysql-cdc/src/test/java/com/ververica/cdc/connectors/mysql/testutils/MySqlContainer.java:36]: https://github.com/ververica/flink-cdc-connectors/blob/b62bb21abfc5bfb83dd181dd84d1a9817686a1ba/flink-connector-mysql-cdc/src/test/java/com/ververica/cdc/connectors/mysql/testutils/MySqlContainer.java#L52

Here, we used the image name mysql/mysql-server:`5.7` or  mysql/mysql-server:`8.0`, so this image can work in Mac with ARM chips.

**Comment by leonardBang** [comment]
> How can retry run UT checks？ @leonardBang

Now we've not robot, could you rebase the latest master? and the rebase will trigger the CI

File: flink-connector-mysql-cdc/src/test/java/com/ververica/cdc/connectors/mysql/testutils/MySqlContainer.java (hunk 1 above)
Comment (whhe) [reply flink-connector-mysql-cdc/src/test/java/com/ververica/cdc/connectors/mysql/testutils/MySqlContainer.java:36]: > No，`mysql/mysql-server:5.7` also can it.
> 
> image

`amd64` aka `x86_64` is not same with `arm64`.

**Comment by yuxiqian** [comment]
Hi @minchowang, Thanks for your contribution! Now MySQL testcases have all been upgraded to 8.0 and they provide native ARM64 images already, so seems #1367 has been resolved. Would you mind if I close this PR or do you have any other comments?

**Pull request closed without merging**